server/src/server.ts

```diff
@@ -1,11 +1,8 @@
 import { App } from "./app";
 import { CalendarController } from "./controllers/calendar.controller";
 
-
 const port: number = 3000;
```

Comment:
Could you change port to `5000`? It conflicts with the frontend port.

Reply:
It's been changed to 3001 in another commit so we shouldn't have the collision anymore